.github/release.yml

```diff
@@ -11,14 +11,17 @@ changelog:
         - major
     - title: New Features & Bugfixes 🎉
       labels:
-        - defects
+        - bug
         - enhancements
-        - feature-request
-    - title: Docs & Maintenance 🔧
+        - feature request
+    - title: Documentation 📖
       labels:
         - docs
+    - title: Maintenance 🔧
+      labels:
         - installation
         - maintenance
         - pre-commit
         - tests
+        - release
```

Comment:
Doesn't matter because everything will be captured here, but maybe useful if we ever refactor this?